pom.xml

```diff
@@ -215,7 +215,7 @@
         <microprofile-healthcheck.version>4.0.1</microprofile-healthcheck.version>
         <microprofile-metrics.version>4.0</microprofile-metrics.version>
         <microprofile-rest-client.version>3.0.1</microprofile-rest-client.version>
-        <microprofile-openapi.version>3.0</microprofile-openapi.version>
+        <microprofile-openapi.version>3.1</microprofile-openapi.version>
```

Comment:
This is already done in FISH-6599

Reply:
git merge will handle it; without this, my PR wouldn't compile :-)